autoapi/mappers/base.py

```diff
@@ -180,7 +181,10 @@ def __init__(self, app, template_dir=None, url_root=None):
         )
 
         def _wrapped_prepare(value):
-            return '\n'.join(prepare_docstring(value))
+            preped_value = '\n'.join(prepare_docstring(value))
+            numpy_value = str(NumpyDocstring(preped_value))
+            final_value = str(GoogleDocstring(numpy_value))
```

Comment:
Google and Numpy docstrings are a Python only thing, so it seems weird to put this here. Also this won't work for the cases where the summary line is not indented. We'd need to pass it into `prepare_docstring` to handle this case.
```
"""This won't work
...
"""
This will
```